pkg/local_object_storage/metabase/db.go

```diff
@@ -144,3 +149,21 @@ func WithPermissions(perm fs.FileMode) Option {
 		c.info.Permission = perm
 	}
 }
+
+// WithBatchSize returns option to specify maximum concurrent operations
+// to be processed in a single transactions. bbolt has no such option,
+// so it is changed right after db is opened.
+func WithBatchSize(s int) Option {
+	return func(c *cfg) {
+		c.boltBatchSize = s
+	}
+}
+
+// WithBatchDelay returns option to specify maximum time to wait before
+// the batch of concurrent transactions is processed.
+// bbolt has no such option, so it is changed right after db is opened.
```

Comment:
lets sync using new line for that disclaimer for both new options

Reply:
Fixed.